themes/ropensci/layouts/partials/head.html

```diff
@@ -1,4 +1,5 @@
 <meta charset="utf-8" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.css">
```

Comment:
my issue in the previous PR was that I used min.css, all.css is needed to actually load the font... I've read it's a recent change.